Sources/HermesAppDelegate.m

```diff
@@ -69,13 +69,9 @@ - (NSMenu *)applicationDockMenu:(NSApplication *)sender {
   NSMenuItem *menuItem;
   Song *song = [[playback playing] playingSong];
   if (song != nil) {
-    [menu addItemWithTitle:@"Now Playing" action:nil keyEquivalent:@""];;
-    [menu addItemWithTitle:[NSString stringWithFormat:@"   %@", [song title]]
-                    action:nil
-             keyEquivalent:@""];
-    [menu addItemWithTitle:[NSString stringWithFormat:@"   %@", [song artist]]
-                    action:nil
-             keyEquivalent:@""];
+    [menu addItemWithTitle:nowPlaying.title action:nil keyEquivalent:@""];;
```

Comment:
Yeah, I suppose I could add back in the indentation (like it's done with iTunes).  Again, just let me know!